v7/navstories/navstories.py

```diff
@@ -38,12 +38,50 @@ def set_site(self, site):
         site.scan_posts()
         # NAVIGATION_LINKS is a TranslatableSetting, values is an actual dict
         for lang in site.config['NAVIGATION_LINKS'].values:
+            # Which paths are navstories active for for lang?
+            navstories_paths = ()
+            if 'NAVSTORIES_PATHS' in site.config and lang in site.config['NAVSTORIES_PATHS']:
+                navstories_paths = site.config['NAVSTORIES_PATHS'][lang]
+
             new = []
+            newsub = {}
             for p in site.pages:
-                if lang in p.translated_to and not p.meta('hidefromnav'):
-                    new.append(p)
-            new_entries = tuple(sorted([(p.permalink(lang), p.title(lang)) for p in new]))
-
+                permalink = p.permalink(lang)
+                for s in navstories_paths:
+                    if permalink.startswith('/' + s + '/'):
```

Comment:
Yes, but next I have to remove `'/' + s + '/'`, I don't know how to do it without regular expressions, and navstories_paths are not regular expressions.